test/Project.toml

```diff
@@ -36,6 +36,7 @@ Unitful = "1986cc42-f94f-5a68-af5c-568840ba703d"
 ArtifactWrappers = "0.2"
 Aqua = "0.8"
 ClimaCoupler = "0.1"
+Dates = "1.9"
```

Comment:
I think we can just have "1" compats for Dates and Statistics since they're in the standard library